qiskit_nature/algorithms/ground_state_solvers/adapt_vqe.py

```diff
@@ -40,13 +43,15 @@
 class AdaptVQE(GroundStateEigensolver):
     """A ground state calculation employing the AdaptVQE algorithm."""
 
+    @deprecate_arguments("0.4.0", {"delta": "gradient"})
```

Comment:
Maybe we can also add a little bit more explanation using the `additional_msg` argument. Kind of like this but with an actual explanation:
```suggestion
    @deprecate_arguments("0.4.0", {"delta": "gradient"}, additional_msg="Here goes my explanation.")
```